lib/index.js

```diff
@@ -35,5 +43,63 @@ Parsely.prototype.initialize = function() {
 };
 
 Parsely.prototype.loaded = function() {
-  return !!window.PARSELY;
+  return !!window.PARSELY.beacon;
+};
+
+/**
+ * Page.
+ *
+ * Only Invoked if dynamicTracking is enabled (otherwise noop)
+ */
+
+Parsely.prototype.page = function(page) {
+  if (!this.options.dynamicTracking) return;
+  var properties = page.properties();
+  var data = {
+    url: page.url(),
+    urlref: page.referrer(),
+    data: properties,
+    js: 1
+  };
+
+  if (this.options.inPixelMetadata) {
```

Comment:
field names are meant to match these: https://www.parsely.com/help/integration/jsonld/